tests/TestCase.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Tests;
+
+use Illuminate\Foundation\Application;
+use Illuminate\Support\ServiceProvider;
+
+class TestCase extends \Orchestra\Testbench\TestCase
+{
+    /**
+     * Get package providers.
+     *
+     * @param  Application  $app
+     * @return array<int, class-string<ServiceProvider>>
+     */
+    protected function getPackageProviders($app): array
+    {
+        return [
+            'KoalaFacade\DiamondConsole\DiamondConsoleServiceProvider',
```

Comment:
Where reference i should read?

Reply:
for package development here https://laravelpackage.com/
for laravel skeleton for package development https://packages.tools/testbench/getting-started/introduction.html

Reply:
got it